### ISSUE TITLE
issue with fullPage.js in Rails 4

### ISSUE THREAD

**Issue by acandael** [alvarotrigo/fullPage.js#986]
Hi, I just integrated fullPage.js in my Rails app, but there seems to be an issue: the second section is stacking up the first second.

I implemented the fullPage.js just for the homepage, as can be seen here:
https://github.com/acandael/adlit-rails/blob/master/app/views/homes/index.html.erb

My fullPage.js implementation was working fine in the static version of this app, as can be viewed here:
http://designer-pig-36111.bitballoon.com

I thought maybe Rails 4 turbolinks was the problem, so I installed the jquery-turbolinks gem, but that didn't solve it.

greetings,

Anthony

**Comment by alvarotrigo** [comment]
Probably your problem is that you are using ajax calls. There are similar topics which were treated in the past, take a look at them:
https://github.com/alvarotrigo/fullPage.js/issues?q=turbolink

**Comment by acandael** [comment]
Hi alvatrigo, thanks for the links. I tried to fix the issue by turning of turbolinks (I put data-no-turbolink in the body element). But that didn't solve the issue.
I checked the console in Chrome Dev Tools, but it doesn't report any Javascript error. The page scroll works when I swipe on my touchpad, but the links on the page don't seem to trigger a pagescroll. Also my layout is messed up, my second section is stacking on top of my first section. On the other hand, the third section is rendered correctly.
